modules/perception/camera/app/cipv_camera.cc

```diff
@@ -754,23 +744,32 @@ bool Cipv::DetermineCipv(const std::vector<base::LaneLine> &lane_objects,
   int32_t old_cipv_index = -1;
   EgoLane egolane_image;
   EgoLane egolane_ground;
+  EgoLane virtual_egolane_ground;
   egolane_ground.left_line.line_point.clear();
   egolane_ground.right_line.line_point.clear();
+  virtual_egolane_ground.left_line.line_point.clear();
+  virtual_egolane_ground.right_line.line_point.clear();
 
   // Get ego lanes (in both image and ground coordinate)
   GetEgoLane(lane_objects, &egolane_image, &egolane_ground, &b_left_valid,
              &b_right_valid);
   ElongateEgoLane(lane_objects, b_left_valid, b_right_valid, options.yaw_rate,
                   options.velocity, &egolane_image, &egolane_ground);
 
+  CreateVirtualEgoLane(options.yaw_rate, options.velocity,
+                       &virtual_egolane_ground);
+
   float min_distance = std::numeric_limits<float>::max();
   float distance;
   for (int32_t i = 0; i < static_cast<int32_t>(objects->size()); ++i) {
     if (debug_level_ >= 2) {
       AINFO << "objects[" << i << "]->track_id: " << (*objects)[i]->track_id;
     }
     if (IsObjectInTheLane((*objects)[i], egolane_image, egolane_ground,
-                          world2camera, &distance)) {
+                          world2camera, &distance) ||
+        IsObjectInTheLane((*objects)[i], egolane_image, virtual_egolane_ground,
+                            world2camera, &distance)
+      ) {
```

Comment:
Put this in the upper row.

Reply:
Thank you for the review! I moved the brackets to the upper row.